runtime/extensions/CMakeLists.txt

```diff
@@ -25,7 +27,7 @@ else()
     set(ENABLE_OPENMP ON)
     set(source_files "StateVectorDynamicCPU.cpp")
 
-    set(LIGHTNING_GIT_TAG "latest_release" CACHE STRING "GIT_TAG value to build Lightning")
+    set(LIGHTNING_GIT_TAG "master" CACHE STRING "GIT_TAG value to build Lightning")
```

Comment:
Yes, They're overridden by the Makefile values. Here, we only specify default values which are pointing to the `master` branch of Lightning, or the `main` branch of Lightning-Kokkos.

Reply:
Ah gotcha, I wasn't sure because I'm not too familiar with CMake and it sounded like set would overwrite the variable, but if the user-specified one takes precedence it's all good :)